OpenSim/Common/Component.h

```diff
@@ -2793,9 +2793,7 @@ void Input<T>::connect(const AbstractOutput& output,
             else {
                 pathName = pathName + "/" + chan.second.getName();
             }
-            if (!annotation.empty() && annotation != chan.second.getChannelName()) {
-                pathName += "(" + annotation + ")";
-            }
+            pathName += "(" + alias + ")";
```

Comment:
I'd prefer to have `()` if no alias has been provided by the user so there is a consistent format between aliased and non-aliased Outputs. I can change (i.e., drop the `()` if `alias.empty()`) if you prefer to have different formats.